tests/qgis-projects/tests/base_layers_user_defined.qgs.cfg

```diff
@@ -1,12 +1,11 @@
 {
     "metadata": {
-        "qgis_desktop_version": 32808,
-        "lizmap_plugin_version_str": "3.15.2-alpha",
-        "lizmap_plugin_version": 31502,
-        "lizmap_web_client_target_version": 30700,
-        "lizmap_web_client_target_status": "Dev",
-        "instance_target_url": "http://localhost:8130/",
-        "instance_target_repository": "intranet",
+        "qgis_desktop_version": 32216,
+        "lizmap_plugin_version_str": "3.14.0",
```

Comment:
> 3.14.0
> "lizmap_web_client_target_version": 30500,

😱 So old, please update your plugin ! 😄

Side note, please generate CFG with the much higher target version as possible.
LWC will not let access this project soon because this CFG is targeting LWC 3.5

Add https://demo.snap.lizmap.com/lizmap_3_7/ in your list of server

Reply:
Resolved